src/index.ts

```diff
@@ -7,9 +7,13 @@ import { parseThresholds } from './parseThresholds.js';
 import { generateFileCoverageHtml } from './generateFileCoverageHtml.js';
 
 const run = async () => {
-  const jsonSummaryPath = path.resolve(core.getInput('json-summary-path'));
-  const jsonFinalPath = path.resolve(core.getInput('json-final-path'));
-  const viteConfigPath = path.resolve(core.getInput('vite-config-path'));
+  // Working directory can be used to modify all default/provided paths (for monorepos, etc)
+  const workingDirectory = core.getInput('working-directory');
+
+  // Paths will be resolved to absolute filepaths from 'path.resolve' call (get defaults from Action inputs)
```

Comment:
I'm not sure if this comment is necessary as it basically just describes what `path.resolve` does - do you mind removing it?

Reply:
Yeah, I can remove it. I had included it originally as I was not aware that `path.resolve` actually resolved right to left, taking the first root path it found. But as you said, this _is_ the behaviour of `path.resolve`...